x-pack/plugins/security_solution/public/management/pages/event_filters/state/index.ts

```diff
@@ -7,12 +7,23 @@
 
 import { ExceptionListItemSchema, CreateExceptionListItemSchema } from '../../../../shared_imports';
 import { AsyncResourceState } from '../../../state/async_resource_state';
+
+export interface EventFiltersPageLocation {
```

Comment:
I had already created this type in `x-pack/plugins/security_solution/public/management/pages/event_filters/types.ts`

Also - how do yo feel about moving the types in this file over to that `types` file? I don't think we currently need this `state` directory.

Reply:
I moved this types to a top level because is used by trusted apps and event filters. I think this can be shareable between them, but if not I'm ok with to move into the event filters type.

Reply:
I'm not sure if its a good for the TA and EventFilters to share the same type for URL route params - they might diverge and have different needs

Reply:
Oh no! I was referring to async_resource_state. So yes, I will move this types to the /types file and remove this state folder

Reply:
Oh yeah, those make since higher up because they will be reused 👍